docs/docs/apps/ui/components.md

```diff
@@ -0,0 +1,4 @@
+---
+title: UI Components
+sidebar_label: UI Components
```

Comment:
So these pages will be empty right? 🤔 We could add the `draft: true` attribute.

Reply:
yes, they will be empty

Reply:
I think we want them to be there, but empty